packages/cli/src/databases/repositories/test-case-execution.repository.ee.ts

```diff
@@ -55,8 +56,12 @@ export class TestCaseExecutionRepository extends Repository<TestCaseExecution> {
 		testRunId: string,
 		pastExecutionId: string,
 		metrics: Record<string, number>,
+		trx?: EntityManager,
 	) {
-		return await this.update(
+		trx = trx ?? this.manager;
```

Comment:
This could also be

```
  trx = this.manager
)
```

Same for the other repository methods here and in `test-run.repository`